### DISCUSSION TITLE
Sugestion for the UI

### DISCUSSION THREAD

**Discussion by DharmaDischarge** [tuskyapp/Tusky#2975] [Ideas]
I wanted to suggest changing 
"Follow" to "Listen to?" and "Following" to "Listening."
and "Unfollow" to "Walk away."
It would make us more aware morally and feel culpable for our engagement.

For example, if someone is spewing hate by presenting it as a question of "Listen to?" I feel it would at least make people think about their choices on Mastodon and remind us of our moral obligation to each other as human beings.

I also feel it aligns with the etiquette of the app during my short time with it.

And it would be easy to implement from what I have been told.

The only downside I can think of is it would confuse users who are used to the current setup.

But the benefits of distinguishing different values from other social media I think outweigh the growing pains.

**Comment by scottiefalkon** [comment]
Way to confusing

**Reply by DharmaDischarge** [reply]
I understand, it was worth a try. and thank you.

**Comment by charlag** [comment]
Hi, I think it's a very interesting idea but we are following mastodon terminology for now I think so we probably shouldn't divert too much.

**Reply by DharmaDischarge** [reply]
Thank you. Do you know if there is a way to submit it to Mastodon directly? I am not very technical and I am new here.

**Reply by charlag** [reply]
Their repo is at https://github.com/mastodon/mastodon
